content/2-running-automation-testing/guide.md

```diff
@@ -0,0 +1,202 @@
+# 2. Running automation test with Kobiton
+
+## 2.1 Prerequisites
+
+* Username 
+1. Go to portal.kobiton.com
+2. In the upper right hand corner, click on your name and in the drop down menu, click on Profile. 
+
+![profile](../../images/2-profile.png)
+
+3. You should see the username. 
+
+![username](../../images/2-user.png)
+
+* API key
+1. Click on your name in the upper righthand corner again and select settings. 
+2. You should be able to find your API key under 'API Keys'. 
+
+![api-key](../../images/2-apikey.png)
+
+* Desired cap
+1. In the navigation bar at the top of the Kobiton website, select Devices. 
+2. Hover over any device and click on the Automation settings button (the gear symbol). 
+
+![automation](../../images/2-automation.png)
+
+3. On the left hand side, you can select your preferred language, and in the right side of the window you will see the desiredCaps. 
```

Comment:
As I saw,  there are much more option that we should focus such as: **App type**, **Devices group**, **Orientation**. I suggest we should mention these field and ask reader to check this carefully because it will change the desiredCaps.